android/sdl_android/src/main/java/com/smartdevicelink/transport/SdlRouterService.java

```diff
@@ -1147,8 +1150,8 @@ public void onCreate() {
     /**
      * The method will attempt to start up the next router service in line based on the sorting criteria of best router service.
      */
-    protected void deployNextRouterService() {
-        List<SdlAppInfo> sdlAppInfoList = AndroidTools.querySdlAppInfo(getApplicationContext(), new SdlAppInfo.BestRouterComparator());
+    protected void deployNextRouterService(VehicleType vehicleType) {
```

Comment:
@kostyaBoss On line 1153 you can remove `VehicleType vehicleType` from the method signature.